crates/holochain/src/local_network_tests.rs

```diff
@@ -35,6 +35,7 @@ use test_case::test_case;
 #[test_case(2)]
 #[test_case(4)]
 #[tokio::test(flavor = "multi_thread")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
```

Comment:
I've looked at this one a bit already. It looks like ops are being received and validated but never correctly integrated so the check for consistency fails. Otherwise, this test appears solid. I have a PR open to make some changes to it already.